src/borg/helpers/manifest.py

```diff
@@ -38,6 +38,7 @@ class Archives(abc.MutableMapping):
     and we can deal with str keys (and it internally encodes to byte keys) and either
     str timestamps or datetime timestamps.
     """
+
```

Comment:
guess this is an accidental change? please remove.

Reply:
yes, removed

Reply:
it actually was the "format on save" setting of vscode